### PR TITLE
add default active styles

### PR DESCRIPTION
**Pull request by jeremy-coleman**
added default active routes/styles using the navigate function and also shows a bit of the flexibility of <switch/>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,16 +17,26 @@
...
-    "node-sass": "^4.9.3",
-    "typescript": "^3.0.1",
-    "uglify": "^0.1.5",
-    "uglify-js": "^3.4.8"
-  },
   "dependencies": {
-    "fuse-react": "^1.0.19",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "deep-equal": "1.0.1",
```
Comment (nchanged) [inline package.json]: why do we need all that?

File: src/app.tsx [hunk 2]
```diff
@@ -42,11 +42,27 @@ class MyRootComponent extends React.Component {
...
+					<Switch config={config}/>
 				</div>
 			</div>
 		);
 	}
 }
 
 ReactDOM.render(<MyRootComponent />, document.querySelector("#root"));
+
+			// alternate/original example without default active styles
```
Comment (nchanged) [inline src/app.tsx:57]: what's the difference with the uncommented part? ;-)

File: package.json (hunk 1 above)
Comment (jeremy-coleman) [reply package.json]: They were required with pnpm, which installs similar to yarn flat, just didnt think to take them out

File: src/app.tsx (hunk 2 above)
Comment (jeremy-coleman) [reply src/app.tsx:57]: ack, fixed

File: package.json (hunk 1 above)
Comment (nchanged) [reply package.json]: please remove them

**Comment by jeremy-coleman** [comment]
updated deps

**Pull request closed without merging**